### ISSUE TITLE
ModuleNotFoundError: No module named 'cryptography'

### ISSUE THREAD

**Issue by ccleberg** [ccleberg/yoshi#1]
When installing using `pipx`, dependencies are not resolved.

See below for the installation, attempt to run, and results.

```bash
pipx install yoshi-cli
yoshi --help

Traceback (most recent call last):
  File "/home/cmc/.local/bin/yoshi", line 5, in <module>
    from yoshi.cli import yoshi
  File "/home/cmc/.local/share/pipx/venvs/yoshi-cli/lib/python3.12/site-packages/yoshi/cli.py", line 8, in <module>
    from yoshi import crypto, database, process
  File "/home/cmc/.local/share/pipx/venvs/yoshi-cli/lib/python3.12/site-packages/yoshi/crypto.py", line 7, in <module>
    from cryptography.fernet import Fernet
ModuleNotFoundError: No module named 'cryptography'
```

**Comment by ccleberg** [comment]
When installing via `pipx`, yoshi is installed into a virtual environment. To solve the dependency issue, run the following:

```bash
pipx inject yoshi-cli cryptography
pipx inject yoshi-cli prettytable
```

**Comment by ccleberg** [comment]
Further resolved by release 0.1.4. Dependencies are now included by default, including the wordlist.
